tf2_client: 1.0.0-1 in 'noetic/distribution.yaml' [bloom]

peci1 opened the pull request:
Increasing version of package(s) in repository `tf2_client` to `1.0.0-2`:

- upstream repository: https://github.com/tpet/tf2_client.git
- release repository: https://gitlab.fel.cvut.cz/cras/ros-release/tf2_client
- distro file: `noetic/distribution.yaml`
- bloom version: `0.11.2`
- previous version for package: `null`

## tf2_client

```
* Adding default server launcher, with install.
* Cmake refactor, namespaced cpp file, dynamic cast to subclass.
* Fix client/server switch in c++ client.
* Passing private node handle as argument.
* C++ client single node handle, clean up, exporting library.
* Log clearing listener and buffer.
* Allow to clear the listener and buffer if they are not needed any more.
* Fix creating duration object.
* Adding cpp support, clean up of parameter names, conditional param reading.
* Passing Duration object to local Buffer instead of float.
* Initial version (untested).
* Contributors: Martin Pecka, Tomas Petricek
```

peci1 committed:
tf2_client: 0.0.0-1 in 'noetic/distribution.yaml' [bloom]

audrow reviewed (commented):
Hi @peci1, the same here goes for https://github.com/ros/rosdistro/pull/37169#discussion_r1191208867.

When you do get access to the repo, could you also add a top level license file?

peci1 committed:
Updated tf2_client to 1.0.0-1

peci1 commented:
I've updated this PR with version 1.0.0.

peci1 commented:
License is in the PR I have to upstream: https://github.com/tpet/tf2_client/pull/1 . And I also added it to the release repo as a patch (although I'm not sure how much value it has there).

quarkytale commented:
Waiting on the license to be added to the source repo, lgtm otherwise!

peci1 committed:
Updated tf2_client to 1.0.0-2

peci1 commented:
I've finally got access to the upstream repo, so I switched to standard release process and dropped the release repo changes. The license file has been added.
